src/test/java/org/jenkinsci/plugins/oic/OicSecurityRealmTest.java

```diff
@@ -153,6 +157,27 @@ void testShouldCheckEscapeHatchWithHashedPassword(JenkinsRule jenkinsRule) throw
         assertFalse(realm.doCheckEscapeHatch(escapeHatchUsername, "wrongPassword"));
     }
 
+    @Test
+    public void testIsValidApiTokenRequest_NoTokenAccessWithoutOicSession(JenkinsRule jenkinsRule) throws Exception {
+        TestRealm realm = new TestRealm.Builder(wireMock).WithMinimalDefaults().build();
+
+        MockHttpServletRequest request = new MockHttpServletRequest();
+        assertFalse(realm.isLogoutRequest(request));
+
+        List<org.springframework.security.core.GrantedAuthority> grantedAuthorities = new ArrayList<>();
+        grantedAuthorities.add(SecurityRealm.AUTHENTICATED_AUTHORITY2);
+        UsernamePasswordAuthenticationToken token =
+                new UsernamePasswordAuthenticationToken("test-user", "", grantedAuthorities);
```

Comment:
I'm confused as to what this is trying to test.  The user.id here is `test-user` but below it is `test`.
but the test is impluing we want to test without an OIDC session, not that the user ids missmatch.